File: src/main/scala/com/github/kazuhito_m/twitterbattler/primitive/domain/model/party/Party.scala
package com.github.kazuhito_m.twitterbattler.primitive.domain.model.party

import com.github.kazuhito_m.twitterbattler.primitive.domain.model.battler.Battler

class Party(var owner: Battler, var members: List[Battler]) extends Serializable {
  def battlers(): List[Battler] = owner :: members
}

File: server/src/main/java/com/github/kazuhito_m/twitterbattler/primitive/domain/model/party/Party.java
package com.github.kazuhito_m.twitterbattler.primitive.domain.model.party;

import com.github.kazuhito_m.twitterbattler.primitive.domain.model.battler.Battler;

import java.util.List;

public class Party {
    final Battler owner;
    final List<Battler> members;

    public Party(Battler owner, List<Battler> members) {
        this.owner = owner;
        this.members = members;
    }
}
